portal/assets/js/ynh_userinfo.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I still don't understand why we need this empty file :/ Can't we just return an empty dict when needed with creating this file x_x (I'm not even sure to understand how/if that file is actually likely to be used somewhere)